### PR TITLE
[15.0][FIX] Update copier template + deprecated method

### PR DESCRIPTION
**Pull request by pedrobaeza**
- Avoid the Flake8 Gitlab problem.
- Remove deprecated method api.Environment.manage().

@Tecnativa

### CODE REVIEW

File: stock_return_request/hooks.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def pre_init_hook(cr):
 
 def post_init_hook(cr, registry):
     """Set moves returnable qty on hand"""
-    with api.Environment.manage():
+    if SUPERUSER_ID:  # Fake if for not having a big diff
```
Comment (rousseldenis) [inline stock_return_request/hooks.py]: @pedrobaeza Is the condition justified for a 'diff' problem ?

Comment (pedrobaeza) [reply stock_return_request/hooks.py]: I think so for avoiding 40 lines of diff (the + and -).

Comment (rousseldenis) [reply stock_return_request/hooks.py]: Yes, but what that matters is the result. And the condition is unwanted

Comment (pedrobaeza) [reply stock_return_request/hooks.py]: OK, as you prefer

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**